e2e/accessibility.ts

```diff
@@ -16,23 +17,12 @@ const MAX_WIDTH = IS_CI ? 80 : process.stdout.columns - 8;
 const THEMES = ['light', 'dark'];
 const AXE_PATH = require.resolve('axe-core');
 const AXE_SOURCE = fs.readFileSync(AXE_PATH, 'utf8');
+const CONCURRENCY = parseInt(process.env.CONCURRENCY as string, 10) || 8;
```

Comment:
I played around with this in CI, `8` seems like the sweet spot of concurrency as higher numbers don't reduce the time and lower numbers increase it. 

With this change, it cuts down total CI time from about 15 minutes to about 5 minutes.